ipa-core/src/error.rs

```diff
@@ -72,6 +72,8 @@ pub enum Error {
     DecompressingInvalidCurvePoint(String),
     #[error(transparent)]
     LengthError(#[from] LengthError),
+    #[error(transparent)]
+    TryFromIntError(#[from] std::num::TryFromIntError),
```

Comment:
This error doesn't seemed to be used anywhere.

Reply:
It's used in breakdown_reveal_aggregation

Reply:
Since we have length error I though it made sense, but I agree it's too generic.